web-console/knikubevirt/cluster-dashboard/cluster-dashboard.md

```diff
@@ -0,0 +1,71 @@
+# Cluster Overview Dashboard
+
+The Cluster Overview dashboard provides a birds-eye view of the cluster's current health, capacity, inventory, utilization, and activity.
+
+The Overview dashboard is the first cluster-level dashboard that appears when the Dashboards nav item is selected. Additional cluster-level dashboards are available as tabs.
+
+![Cluster Overview Dashboard Healthy](img/1-0-cluster-healthy.png)
```

Comment:
I guess we should drop the Monitoring and Hardware tabs from these overall views...

Reply:
Good point, fix coming.